devito/tools/data_structures.py

```diff
@@ -43,10 +43,15 @@ class EnrichedTuple(tuple, Pickable):
     A tuple with an arbitrary number of additional attributes.
     """
 
+    __rargs__ = ('*values',)
```

Comment:
keep consistent with signature so rename as items or change to values in __new__

Reply:
seconded, this should be called "items" though in practice it might work the same since it's prefixed with "*"